core/domain/stats_domain.py

```diff
@@ -1099,23 +1232,25 @@ class StateAnswers:
     """Domain object containing answers submitted to an exploration state."""
 
     def __init__(
-            self, exploration_id, exploration_version, state_name,
-            interaction_id, submitted_answer_list,
-            schema_version=feconf.CURRENT_STATE_ANSWERS_SCHEMA_VERSION):
+            self, exploration_id: str, exploration_version: int,
+            state_name: str, interaction_id: str,
+            submitted_answer_list: List[SubmittedAnswer],
+            schema_version: int=
+            feconf.CURRENT_STATE_ANSWERS_SCHEMA_VERSION) -> None:
```

Comment:
Instead of breaking here, move `feconf.CURRENT_STATE_ANSWERS_SCHEMA_VERSION` to the end of the previous line, and then break before the `)` (which I think you can deindent to match the indentation level of the opening paren).

Reply:
Done